oslo_vmware/rw_handles.py

```diff
@@ -313,6 +315,27 @@ def _log_progress(self, progress):
             LOG.debug("Data transfer progress is %d%%.", progress)
             self._last_logged_progress = progress
 
+    def _check_progress_stalled(self, progress, now):
+        """Check if we still make progress
+
+        Compares the given progress to the last known progress to see if we
+        still go forward. Emits a warning otherwise.
+        """
+        if progress < self._last_progress:
```

Comment:
Just for me to understand - when is this likely to happen ? How will a progress go backward ?

Reply:
:D never, I hope. I wouldn't expect it to. It shouldn't. I'd even say, it can't. So yeah, might be dead code in the end, but if this ever happens, I'd like to know ;)